crash_similarity.py

```diff
@@ -20,6 +21,7 @@
 from download_data import download_stack_traces_for_signature
 
 pyximport.install()
+logging.basicConfig(level=logging.DEBUG, format='%(asctime)s - %(levelname)s - %(message)s')
```

Comment:
Actually, let's remove the `basicConfig` calls altogether. We will use the default logging format and change it if needed in the future.